internal/cloud/realm/client.go

```diff
@@ -72,6 +72,7 @@ type Client interface {
 
 	Templates() ([]Template, error)
 	ClientTemplate(groupID, appID, templateID string) (*zip.Reader, error)
+	CompatibleTemplates(groupID, appID string) ([]Template, error)
```

Comment:
`Templates` _is_ fetching all templates, and `CompatibleTemplates` is a filtered list of templates that are compatible with the app. They both return only 
```
        ID   string `json:"id"`
	Name string `json:"name"`
```
`ClientTemplate` is the most granular of the endpoints, which fetches 
 everything about the template specified from `templateID`, and for realm-cli returns a zip package containing the template information

Reply:
Yeah, I'm cool with renaming `Templates` to be `AllTemplates` to make that more clear. 

I think it should be relatively apparent that the endpoints return the same data given the interface definition

Reply:
cool, and just to clarify I think that also means we can then remove `Compatible` from this new function name?

Reply:
@andrewzlchen I'm iffy from making this just `Templates` because the templates that we fetch for this app aren't necessarily the templates that this app is currently using; they're _potential_ templates that the app _could_ use because they're compatible

Reply:
yeah i think we should call them:

`AllTemplates` and `CompatibleTemplates`

If `CompatibleTemplates` is not preferable, maybe we can call them `SiblingTemplates`, `SupportedTemplates` or something similar

Reply:
fine with it as `All`/`Compatible` here